src/common/range_util.cc

```diff
@@ -17,61 +17,6 @@
 #include "knowhere/log.h"
 namespace knowhere {
 
-///////////////////////////////////////////////////////////////////////////////
-// For Faiss index types
-size_t
-CountValidRangeSearchResult(const faiss::RangeSearchResult& res, const bool is_ip, const int64_t nq, const float radius,
-                            const float range_filter, size_t*& lims) {
-    lims = new size_t[nq + 1];
-    lims[0] = 0;
-    for (int64_t i = 0; i < nq; i++) {
-        int64_t valid = 0;
-        for (size_t j = res.lims[i]; j < res.lims[i + 1]; j++) {
-            if (distance_in_range(res.distances[j], radius, range_filter, is_ip)) {
-                valid++;
-            }
-        }
-        lims[i + 1] = lims[i] + valid;
-    }
-    return lims[nq];
-}
-
-void
-FilterRangeSearchResultForOneNq(const int64_t i_size, const float* i_distances, const int64_t* i_labels,
-                                const bool is_ip, const float radius, const float range_filter, const int64_t o_size,
-                                float* o_distances, int64_t* o_labels, const BitsetView& bitset) {
-    int64_t num = 0;
-    for (int64_t i = 0; i < i_size; i++) {
-        auto dis = i_distances[i];
-        auto id = i_labels[i];
-        KNOWHERE_THROW_IF_NOT_MSG(bitset.empty() || !bitset.test(id), "bitset invalid");
-        if (distance_in_range(dis, radius, range_filter, is_ip)) {
-            o_labels[num] = id;
-            o_distances[num] = dis;
-            num++;
-        }
-    }
-    KNOWHERE_THROW_IF_NOT_FMT(num == o_size, "%" SCNd64 " not equal %" SCNd64, num, o_size);
-}
-
-void
-GetRangeSearchResult(const faiss::RangeSearchResult& res, const bool is_ip, const int64_t nq, const float radius,
```

Comment:
this API is used in flat.cc and ivf.cc, please double check

Reply:
the `GetRangeSearchResult(const std::vector<std::vector<float>>& result_distances, const std::vector<std::vector<int64_t>>& result_labels` variant is used there